packages/build-utils/src/fs/run-user-scripts.ts

```diff
@@ -392,6 +372,34 @@ export async function walkParentDirs({
   return null;
 }
 
+async function walkParentDirs2({
```

Comment:
This is similar to `walkParentDirs` above but `walkParentDirs2` supports an array of filenames and returns an array of filenames. I think we could eventually merge the two but since `walkParentDirs` is exported publicly, I didn't want to change it.